taxonomy-editor-frontend/src/pages/project/editentry/index.tsx

```diff
@@ -23,6 +25,21 @@ const EditEntry = ({ taxonomyName, branchName, id }: EditEntryProps) => {
   const [openDeleteDialog, setOpenDeleteDialog] = useState(false);
   const [openSuccessDialog, setOpenSuccessDialog] = useState(false);
 
+  const url = createURL(taxonomyName, branchName, id);
+
+  const { data: node } = useQuery({
+    queryKey: [url],
+    queryFn: async () => {
+      const response = await fetch(url);
+      if (!response.ok) {
+        throw new Error("Failed to fetch node");
+      }
+      return response.json();
+    },
+  });
+
+  const isExternalNode = node?.is_external === true;
```

Comment:
nit: since you determine here if a node is external or not, I feel it may be clearer to just pass down a `isEditable` prop to the children (no need to access `node?.isExternal` anywhere else)